lib/nexpose/util.rb

```diff
@@ -54,18 +50,15 @@ module HostOrIP
     # @return [IPRange|HostName] Valid class, if it can be converted.
     #
     def convert(asset)
-      begin
```

Comment:
Using the `begin` keyword is not required in `def/end` clauses since they have their own scope.